doc/02-installation.md

```diff
@@ -125,14 +125,15 @@ wget https://packages.icinga.com/subscription/rhel/ICINGA-release.repo -O /etc/y
 If you are using RHEL you need to additionally enable the `codeready-builder`
 repository before installing the [EPEL rpm package](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
 
-#### RHEL 8
+#### RHEL 8 or Later
 
 ```bash
 ARCH=$( /bin/arch )
+OSVER=$( cat /etc/redhat-release | grep -Po '[0-9](?=\.)' )
```

Comment:
That wouldn't work on RHEL 10 as it matches only a single digit.

Also, that file doesn't look like the best choice for parsing, given that this is it's content:
```
Red Hat Enterprise Linux release 9.0 (Plow)
```

I'd go with the following instead, similar to how we already do it on Fedora:
```bash
$(. /etc/os-release; echo "${VERSION_ID%%.*}")
```

Reply:
Thanks for the feedback! I've implemented that with the last commit. 
Any other suggestions or can I rebase this branch?